.gitignore

```diff
@@ -1,2 +1,2 @@
 db.sqlite3
-*pyc
+*.pyc
```

Comment:
How about using [https://github.com/github/gitignore/blob/master/Python.gitignore](this boilerplate) or at least changing it to:
```
# regular python
__pycache__/
*.py[cod]
*$py.class
# django
*.log
.static_storage/
.media/
local_settings.py
```

Reply:
ok